### ISSUE TITLE
Batch mode cannot be executed from config file

### ISSUE THREAD

**Issue by DaniDelHoyo** [ccsb-scripps/AutoDock-Vina#199]
Seems like when the batch input mode is specified from the config file, vina (as installed from conda, AutoDock Vina 58016d8-mod) does not recognize the batch option and tries to read the batch regular expression (input_dir/*.pdbqt) as a file, resulting in an error. I can always specify the batch option from the command line, so it is not a big deal, but still might be worth check in.

------ Command -------- 

vina --config configFile.txt

------ Config file --------  

batch = ./tmp/*.pdbqt
scoring = ad4
maps = 1s63_tz
center_x = 11.148385089189023
center_y = 125.14929904879297
center_z = 1.9764267131155862
size_x = 70.0
size_y = 70.0
size_z = 70.0
dir = .
cpu = 4
exhaustiveness = 3
max_evals = 0
num_modes = 2
min_rmsd = 2.0
verbosity = 2

----- Error ------ 

Scoring function : ad4
Exhaustiveness: 3
CPU: 4
Verbosity: 2

Reading AD4.2 maps ... done.

Error: could not open "./tmp/*.pdbqt" for reading.

----- Alternative ------ 

*Remove batch line from config file
Command:  vina --batch ./tmp/*.pdbqt --config configFile.txt

**Comment by iwatobipen** [comment]
How about use absolute path instead of relative path for batch and dir argument in your config file.
It will work fine.
Best,
Taka

**Comment by rwxayheee** [comment]
Added to Maintenance Release. We will look at conda-forge feedstock and whether this is caused by a different patch version, or a bug we can fix.

**Comment by rwxayheee** [comment]
The current behavior is that the path in config for batch ligand files doesn't expand. 

Still, in command-line, it's possible to do: 

```
vina --batch ./ligands/*.pdbqt
```

Additionally, now in develop it's supported, after #384 

```
vina --batch ligands
```

The config equivalence should work the same way, if we want to includes all *.pdbqt files in folder ligands

Closing as won't fix for now, but you're welcome to submit a new PR to enable the feature
